src/containers/edit-profile/professional-info-tab/ProfessionalInfoTab.tsx

```diff
@@ -118,6 +118,12 @@ const ProfessionalInfoTab: FC = () => {
     </Box>
   )
 
+  const professionalTabCategoriesDescription = t(
+    userRole === UserRoleEnum.Student
+      ? 'editProfilePage.profile.professionalTab.categoriesDescriptionForStudent'
+      : 'editProfilePage.profile.professionalTab.categoriesDescriptionForTutor'
+  )
+
```

Comment:
This code can be simplified. You can access to the translation by the exact role. For example `editProfilePage.profile.professionalTab.categoriesDescription.${userRole}`, so you can delete this code and just use this path instead of the `professionalTabCategoriesDescription` variable